packages/cli/generators/controller/index.js

```diff
@@ -257,6 +258,38 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
       default:
         break;
     }
+
+    this.artifactInfo.exclude = [];
+    if (this.artifactInfo.idOmitted) {
+      this.artifactInfo.exclude.push(this.artifactInfo.id);
+    }
+
+    const project = new relationUtils.AstLoopBackProject();
+    const fileName = path.resolve(
+      this.artifactInfo.modelDir,
+      utils.getModelFileName(this.artifactInfo.modelName),
+    );
+    const modelFile = project.addSourceFileAtPath(fileName);
+    const modelClass = modelFile.getClassOrThrow(this.artifactInfo.modelName);
+    for (const classProperty of modelClass.getInstanceProperties()) {
+      for (const decorator of classProperty.getDecorators()) {
+        for (const decoratorArg of decorator.getArguments()) {
+          if (decoratorArg.getProperty) {
+            const readOnlyProperty = decoratorArg.getProperty('readOnly');
+            if (readOnlyProperty) {
+              const readOnlyValue = readOnlyProperty
+                .getInitializerOrThrow()
+                .getText();
+              if (readOnlyValue === '1' || readOnlyValue === 'true') {
+                console.log(classProperty.getName());
```

Comment:
Is this console log intentional ?

Reply:
No. This was there by mistake. Removed it.